extensions/interactions/ImageClickInput/directives/ImageClickInput.js

```diff
@@ -28,7 +28,8 @@ oppia.directive('oppiaInteractiveImageClickInput', [
     return {
       restrict: 'E',
       scope: {
-        onSubmit: '&'
+        onSubmit: '&',
+        currentCard: '='
```

Comment:
Use '&' here, you don't need this to be two-way, I think.

Reply:
Done